calendar-be/PlannerApp/src/main/java/com/PlannerApp/PlannerApp/PlannerAppApplication.java

```diff
@@ -1,10 +1,12 @@
 package com.PlannerApp.PlannerApp;
 
+import org.mybatis.spring.annotation.MapperScan;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@MapperScan("com.PlannerApp.PlannerApp.Repositories")
```

Comment:
whats this used for?

Reply:
Spring Boot auto-creates mapper implementations as beans, allowing you to use @Autowire to access them in your services for database interactions.